src/widget/wwidgetgroup.cpp

```diff
@@ -215,5 +215,6 @@ int WWidgetGroup::getHighlight() const {
 void WWidgetGroup::setHighlight(int highlight) {
     m_highlight = highlight;
     update();
+    style()->polish(this);
```

Comment:
Yea, that WPushButton one should probably be an update, not a repaint (we typically only use repaint for things that update on the waveform timer). I meant add an unpolish here.